docker/prod/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM python:3.7-alpine3.10 AS base
+
+ENV PYTHONUNBUFFERED=0
+
+RUN apk add --no-cache --virtual common-deps curl
+
+FROM base AS builder
+
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python \
+    && ln -s /root/.poetry/bin/poetry /usr/local/bin/poetry
+
+COPY . /tmp/build
+
+WORKDIR /tmp/build
+
+RUN set -x \
+    && poetry export -f requirements.txt > requirements.txt
+
+
+FROM base AS installer
+
+COPY ./docker/common/install_build_deps /tmp/build
```

Comment:
the file is called `install_build_deps.sh` (as seen three lines below) and builds fail on this.